horace_core/sqw/@DnDBase/save_xye.m

```diff
@@ -0,0 +1,56 @@
+function  save_xye(obj,varargin)
+% Save 1D,2D,.. sqw or dnd object to ascii file.
```

Comment:
method in dndbase not sqwndndbase

Reply:
as above, `sqw` method delegates job to `dnd`.  sqw method defined in `sqw` class itself so is not very suitable for long description